### PR TITLE
Include Marcel as TSC member

### PR DESCRIPTION
**Pull request by tsteenbe**

### CODE REVIEW

**Comment by sschuberth** [comment]
Typo in your configured Git email address: `From: Thomas Steenbergen <thomas_steenbergen@epam>`

**Comment by sschuberth** [comment]
Now you have a typo in "thmas_steenbergen@epam.com" 😉

**Review by fviernau — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ All meetings in person or online are held under [OSS Review Toolkit's charter](C
 You can find the meeting minutes in the [minutes directory](./minutes/) in this repository.
 
 Unless otherwise indicated the below names in the minutes refer to the following persons:
-- *Frank*: Frank Viernau ([@fviernau](https://github.com/fviernau)])
+- *Frank*: Frank Viernau ([@fviernau](https://github.com/fviernau))
```
Comment (sschuberth) [inline README.md:22]: Ideally this fix should go to a separate commit.

Comment (sschuberth) [reply README.md:22]: Tired of waiting for @tsteenbe to make this fix, so let's get this merged as I need to refer to the current TSC members 😉

**Review by sschuberth — APPROVED** [review]
